client/app/scripts/superdesk-search/search.js

```diff
@@ -1215,6 +1215,17 @@
                         multiEdit.create(multi.getIds());
                         multiEdit.open();
                     };
+
+                    scope.createPackage = function() {
+                        packages.createPackageFromItems(multi.getItems())
+                        .then(function(new_package) {
+                            superdesk.intent('author', 'package', new_package);
+                        }, function(response) {
+                            if (response.status === 403 && response.data && response.data._message) {
```

Comment:
I agree with that, we need to revise the error codes and make them consistent across the application.